packages/cli/package.json

```diff
@@ -37,6 +37,7 @@
     "connect": "^3.6.5",
     "cosmiconfig": "^5.1.0",
     "deepmerge": "^3.2.0",
+    "didyoumean": "^1.2.1",
```

Comment:
Did you run `yarn` so that `yarn.lock` is updated?

Reply:
I use `npm` and always miss it out!